### PR TITLE
Create Python Solutions (And fix existing ones) for Bovine Genomics

### PR DESCRIPTION
**Pull request by Aadit-Ambadkar**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [X] I have tested my code.
- [X] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [X] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-736.mdx [hunk 1]
```diff
@@ -170,6 +170,39 @@ public class BovineGenomics {
...
+
+for i in range(n):
+	spotted[i] = fin.readline().strip()
+
+for i in range(n):
+	plain[i] = fin.readline().strip()
+
+ans = 0
+for i in range(m):
+	a = set()
```
Comment (SansPapyrus683) [inline solutions/usaco-736.mdx:191]: better name

File: solutions/usaco-736.mdx [hunk 2]
```diff
@@ -170,6 +170,39 @@ public class BovineGenomics {
 ```
 
 </JavaSection>
+<PySection>
+```py
+fin = open("cownomics.in", "r")
+fout = open("cownomics.out", "w")
```
Comment (SansPapyrus683) [inline solutions/usaco-736.mdx:176]:
```suggestion
```

File: solutions/usaco-736.mdx [hunk 3]
```diff
@@ -170,6 +170,39 @@ public class BovineGenomics {
...
+		a.add(plain[j][i])
+	
+	for j in range(n):
+		if spotted[j][i] in a:
+			break
+	else:  # We never broke out of the above loop
+		ans += 1
+	
+fout.write(str(ans))
+fout.close()
```
Comment (SansPapyrus683) [inline solutions/usaco-736.mdx:203]:
```suggestion
print(ans, file=open('cownomics.out', 'w'))
```

File: solutions/usaco-736.mdx [hunk 4]
```diff
@@ -170,6 +170,39 @@ public class BovineGenomics {
 ```
 
 </JavaSection>
+<PySection>
+```py
+fin = open("cownomics.in", "r")
```
Comment (SansPapyrus683) [inline solutions/usaco-736.mdx:175]:
```suggestion
fin = open('cownomics.in', 'r')
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
p good

File: solutions/usaco-736.mdx (hunk 1 above)
Comment (Aadit-Ambadkar) [reply solutions/usaco-736.mdx:190]: which?

Comment (SansPapyrus683) [reply solutions/usaco-736.mdx:190]: `a`

**Review by envyaims — APPROVED** [review]
a

**Comment by Aadit-Ambadkar** [comment]
@SansPapyrus683 anything else?

**Pull request merged**